api/v1beta1/hyperconverged_types.go

```diff
@@ -387,6 +387,10 @@ type VirtualMachineOptions struct {
 // by default yet.
 // +k8s:openapi-gen=true
 type HyperConvergedFeatureGates struct {
+	// Allow to expose a limited set of host metrics to guests.
+	// +optional
+	DownwardMetrics *bool `json:"downwardMetrics,omitempty"`
```

Comment:
Please add default value

Reply:
We don't want to have a default value here right now.